Assignment2/Assignment2_QNo3/SolAssignmentNo2_QNo3.rb

```diff
@@ -0,0 +1,31 @@
+# => 3 Write an example of how you can use super method with and without arguments.
+# => Super Parent Class
+class SuperParentSkeleton
+  def initialize
+    puts "Hii You chose option : Super without Arguments in Parent Class"
+  end
+
+  def super_argument(num)
+    puts "hii You chose option : Super with Arguments in Parent Class"
+  end
+end
+
+# => Super child class
+class SuperChildSkeleton < SuperParentSkeleton
+  def initialize
+    # super without arguments
+    super
+    puts "Hii You chose option : Super without Arguments in Child Class"
+  end
+
+  def super_argument(num)
+    # super with arguments
+    super(num)
+    puts "Hii You chose option : Super with Arguments in Child Class"
+    end
```

Comment:
@HrisikeshSinghaChowdhury wrong alignment ! Also did you check by calling super without argument from super_argument method in child class ?

Reply:
yes checked